web/src/message_list_view.js

```diff
@@ -219,6 +219,7 @@ function populate_group_from_message_container(group, message_container) {
         group.topic = message_container.msg.topic;
         group.match_topic = util.get_match_topic(message_container.msg);
         group.stream_url = message_container.stream_url;
+        group.archived = stream_data.is_archived(message_container.msg.stream_id);
```

Comment:
Let's call this `is_stream_archived` -- we should scope it clearly as not a "archived message".

Reply:
Done.